.github/workflows/pr-loadtest.yml

```diff
@@ -0,0 +1,64 @@
+name: Manual Load Test
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'Pull Request Number'
+        required: true
+
+env:
+  IMG: europe-docker.pkg.dev/kyma-project/dev/telemetry-manager:PR-${{ github.event.inputs.pr_number }}
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "greet"
+  load-test:
+    strategy:
+      max-parallel: 1
+      matrix:
+        k8s_version: [ 1.27, 1.28 ]
+        argument: ["-t traces", "-t traces -m true", "-t traces -b true", "-t metrics", "-t metrics -m true", "-t metrics -b true"]
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          ref: 'refs/pull/${{ github.event.inputs.pr_number }}/head'
+          repository: ${{ github.repository }}
+
+      - name: Setup Golang
+        uses: "./.github/template/setup-golang"
+
+
+      # save gardener kubeconfig to a temp file in order to pass it to the command
+      - name: Save service account to file
+        shell: bash
+        run: 'echo "$GARDENER_SA" > /tmp/gardener-sa.yaml'
+        env:
+          GARDENER_SA: ${{ secrets.GARDENER_SA }}
+
+      # provision cluster and run tests
+      - name: Provision Gardener
+        run: make provision-gardener
+        env:
+          GARDENER_SECRET_NAME: ${{ secrets.GARDENER_SECRET_NAME }}
+          GARDENER_PROJECT: ${{ secrets.GARDENER_PROJECT }}
+          GARDENER_SA_PATH: /tmp/gardener-sa.yaml
+          GARDENER_K8S_VERSION: ${{ matrix.k8s_version }}
```

Comment:
Think about how the clusters should be named unique and make that configurable instead, k8s version is not relevant for load tests I assume

Reply:
Cluster name is generated based on the `-n` flag. The `-n` flag sets the test name and same name would be used to call the cluster name.